cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/ClouderaManagerRepo.java

```diff
@@ -71,13 +90,9 @@ public ClouderaManagerRepo withGpgKeyUrl(String gpgKeyUrl) {
         return this;
     }
 
-    public Map<String, Object> asMap() {
```

Comment:
is it not used anymore?

Reply:
I could not find any reference for it, it was introduced here and not used then either: https://github.com/hortonworks/cloudbreak/commit/b05fcdc058fd970cf9250f706b058984b3f3ff02